elao.app.docker/README.md

```diff
@@ -806,6 +808,26 @@ In  order for https to work properly, you must:
 
 3. For firefox only, browse to `about:config` and ensure `security.enterprise_roots.enabled` value is set to true
 
+### Ubuntu
+
+1. ensure elao ca certificate has been added to your local keychain (one time for *all* projects)
+
+   ```shell
+    $ sudo cp .manala/certificates/ca.crt /usr/local/share/ca-certificates
+    $ sudo update-ca-certificates
+    ```
+
+2. generate a project certificate (one time *by* project, remember to commit them right after)
+
+    ```shell
+    ⇒  make provision.certificates
+    ```
+
+3. For Chrone and Firefox, add .manala/certificates/ca.crt certificate on Security and Privacy.
```

Comment:
Peut-être inverser les points 2 et 3, et préciser `(one time for *all* projects)` sur le point 2 (anciennement 3)
Et peut-être en profiter pour faire la même chose coté macOs